content-scope-scripts/content-scope-scripts-impl/src/main/java/com/duckduckgo/contentscopescripts/impl/ContentScopeScriptsJsInjectorPlugin.kt

```diff
@@ -39,7 +40,11 @@ class ContentScopeScriptsJsInjectorPlugin @Inject constructor(
         }
     }
 
-    override fun onPageFinished(webView: WebView, url: String?, site: Site?) {
+    override fun onPageFinished(
```

Comment:
I think @mikescamell made some changes to avoid all these noise in the PRs? Is all these formatting needed?

Reply:
It's already happened before his fix so I think I'd need to manually revert these formatting changes.